akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/RetentionCriteria.scala

```diff
@@ -32,6 +32,9 @@ object RetentionCriteria {
    *
    * Use [[SnapshotCountRetentionCriteria.withDeleteEventsOnSnapshot]] to
    * delete old events. Events are not deleted by default.
+   *
+   * If multiple events are persisted with a persist the snapshot will happen after
```

Comment:
```suggestion
   * If multiple events are persisted together the snapshot will happen after
```

"persisted with a persist" -> this sounds strange

Reply:
I had initially suggested "together" to later see that the next change was using "a single Effect". That's better.

Reply:
@renatocaval I think you meant this?
```suggestion
   * If multiple events are persisted with a single Effect, the snapshot will happen after
```

Reply:
Yes, indeed.
We got concurrent access issues here.